### PR TITLE
[RFC] inccommand: Ignore leading modifiers in the command

### PR DESCRIPTION
**Pull request by jamessan**
@wilywampa suggested this in gitter.  I still need to write the tests for it.

### CODE REVIEW

File: src/nvim/ex_docmd.c [hunk 1]
```diff
@@ -9860,6 +9860,12 @@ bool cmd_can_preview(char_u *cmd)
     return false;
```
Comment (justinmk) [inline src/nvim/ex_docmd.c]: The doc could also be changed to:

    /// @param[in] cmd Commandline to check. May start with a range or modifier.

Comment (jamessan) [reply src/nvim/ex_docmd.c]: Doc updated and tests added.

**Review by justinmk — APPROVED** [review]

File: src/nvim/ex_docmd.c [hunk 2]
```diff
@@ -9883,7 +9883,7 @@ static void ex_terminal(exarg_T *eap)
 
 /// Checks if `cmd` is "previewable" (i.e. supported by 'inccommand').
 ///
-/// @param[in] cmd Commandline to check. May start with a range.
+/// @param[in] cmd Commandline to check. May start with a range or modifer.
```
Comment (joereynolds) [inline src/nvim/ex_docmd.c]: Small thing but 'modifer' should be 'modifier'

Comment (jamessan) [reply src/nvim/ex_docmd.c]: Good catch.  I'll fix that tonight.

**Comment by wilywampa** [comment]
What about user-created modifiers?

**Comment by jamessan** [comment]
> What about user-created modifiers?

I'm not sure what you mean here, but there's nothing preventing continuing to improve the inccommand support.

**Pull request merged**